### DISCUSSION TITLE
Issue in downloading the virtual machine

### DISCUSSION THREAD

**Discussion by LCaranzi** [robotology/community#193] [Q&A] [answered]
Dear all,

I had troubles in downloading the VM. I managed to connect to the server (i.e. `ftp.iit.it`), but I do not have access to the subdirectory of the server, thus I cannot download the file `iCub_Ubuntu_16.04_x64.zip`.

The error message is **"Failed to retrieve directory listing"** as it can be seen in the image below.

Can someone help me?
Many thanks.

![directoryerror](https://cloud.githubusercontent.com/assets/26228175/23619470/04249130-0294-11e7-9fec-8b57e78cf497.jpg)

**Comment by pattacini** [comment]
Hi @mbrunettini 
Just for your reference, I've tried to download the VM via FileZilla and I was successful.

Could you help @LCaranzi?

**Comment by mbrunettini** [comment, marked as answer]
You must use the FTPS protocol (port 990), it seems that you connected using the default FTP port.

Ref. to [these instructions](https://github.com/vvv-school/vvv17/wiki/Before%20arriving%20at%20VVV#download-our-virtual-machine).

**Comment by LCaranzi** [comment]
Ok thanks, it worked!
